src/game_updater/GameUpdater.cpp

```diff
@@ -8,6 +8,7 @@
 #include "helpers/Logging.h"
 #include "helpers/TimingConstants.h"
 
+#define MAX_DELTA_Y 0.007f
```

Comment:
Could you give this a better name and move it to GameCamera.h along with CAMERA_X_SPACING and stuff?